caret.repos

```diff
@@ -27,3 +27,8 @@ repositories:
     type: git
     url: https://github.com/tier4/ros2caret.git
     version: main
+  # Dependency for CI/CD
+  CARET/caret_common:
+    type: git
+    url: https://github.com/autowarefoundation/caret_common.git
```

Comment:
I think caret_common is under tier4 account.

```suggestion
    url: https://github.com/tier4/caret_common.git
```

Reply:
@hsgwa  
Thank you for comment. Exactly yes, it's my mistake.

Reply:
fixed in
https://github.com/tier4/caret/pull/27/commits/3dea77304a682598175ec559f7a60bbeaccd9779